src/NuGet.Core/NuGet.Build.Tasks/BuildTasksUtility.cs

```diff
@@ -621,7 +620,7 @@ private static IEnumerable<Packaging.PackageReference> GetInstalledPackageRefere
             {
                 try
                 {
-                    XDocument xDocument = XmlUtility.Load(projectConfigFilePath);
+                    XDocument xDocument = Shared.XmlUtility.Load(projectConfigFilePath);
```

Comment:
You are already importing shared. 
You can clean-up the imports.

Reply:
Used static import to simplify the qualifier in the latest commit.